site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -12,6 +12,38 @@ const merged1 = 'Merged Thread Title: '.concat(title3, '\n\n', content3);
 const merged2 = 'Merged Thread Title: '.concat(title2, '\n\n', content2);
 const attachment1 = 'sea_animals.png';
 
+const goToForum = (user) => {
```

Comment:
I'm not sure if this needs to be a new function, as it's just a couple of lines of code, but it isn't a blocker from being merged.